packages/cli/commands/upload.js

```diff
@@ -34,7 +35,7 @@ const { i18n } = require('@hubspot/cli-lib/lib/lang');
 const i18nKey = 'cli.commands.upload';
 const { EXIT_CODES } = require('../lib/enums/exitCodes');
 
-exports.command = 'upload <src> <dest>';
+exports.command = 'upload [--src] [--dest]';
```

Comment:
Yargs allows you to pass the name of a positional arg when running the command so we don't need to (also, can't?) add explicit options here.